frontend/apps/picasso/pages/terms-of-use.tsx

```diff
@@ -0,0 +1,101 @@
+import { GoverningLawAndVenue } from "@/components/Organisms/Static/TermsOfUse/GoverningLawAndVenue";
+import { AccessAndAcceptance } from "@/components/Organisms/Static/TermsOfUse/AccessAndAcceptance";
+import { EntireAgreement } from "@/components/Organisms/Static/TermsOfUse/EntireAgreement";
+
+export default function TermsOfUse() {
+  return (
+    <DefaultLayout>
+      <Grid
+        container
+        sx={{ mx: "auto", textAlign: "justify" }}
+        maxWidth="58ch"
+        rowSpacing={5}
+        columns={10}
+        direction="column"
+        justifyContent="center"
+        gap={4}
+      >
+        <Grid item xs={12} mt={9}>
+          <article>
+            <header>
+              <PageTitle title={"Terms of use"} subtitle="January 17, 2023" />
+            </header>
+            <Preface />
+            <List
+              sx={{
+                listStylePosition: "inside",
+                padding: 0,
+              }}
+            >
+              <ListItem>Eligibility</ListItem>
+              <Eligibility />
+              <ListItem>Informational Resource</ListItem>
+              <InformationalResource />
+              <ListItem>Intellectual Property Rights</ListItem>
+              <IntellectualPropertyRights />
+              <ListItem>Third-Party Website and Content</ListItem>
+              <ThirdPartyWebsiteAndContent />
+              <ListItem>Unacceptable Use or Conduct</ListItem>
+              <UnacceptableUseOrConduct />
+              <ListItem>Forward-Looking Statements</ListItem>
+              <ForwardLookingStatements />
+              <ListItem>Use Cases</ListItem>
+              <UseCases />
+              <ListItem>Not an Offering</ListItem>
+              <NotAnOffering />
+              <ListItem>Not Professional Advice</ListItem>
+              <NotProfessionalAdvice />
+              <ListItem>Assumption of Risks</ListItem>
+              <AssumptionOfRisks />
+              <ListItem>No Warranties</ListItem>
+              <NoWarranties />
+              <ListItem>Staking</ListItem>
+              <Staking />
+              <ListItem>Limitation of Liability</ListItem>
+              <LimitationOfLiability />
+              <ListItem>Indemnification and Full Release</ListItem>
+              <IndemnificationAndFullRelease />
+              <ListItem>Dispute Resolution</ListItem>
+              <DisputeResolution />
+              <ListItem>Indemnification</ListItem>
+              <Indemnification />
+              <ListItem>Reserved Rights</ListItem>
+              <ReservedRights />
+              <ListItem>Assignment</ListItem>
+              <Assignment />
+              <ListItem>Governing Law and Venue</ListItem>
+              <GoverningLawAndVenue />
+              <ListItem>Entire Agreement</ListItem>
+              <EntireAgreement />
+              <ListItem>Access and Acceptance</ListItem>
+              <AccessAndAcceptance />
```

Comment:
I don't really understand the reasoning behind splitting these into a separate file

Reply:
Will bring them back into a single file. thanks!